docs/guides/setup.md

```diff
@@ -14,8 +14,8 @@ You can download the Video.js source and host it on your own servers, or use the
 
 ### CDN Version ###
 ```html
-<link href="//vjs.zencdn.net/4.12/video-js.min.css" rel="stylesheet">
-<script src="//vjs.zencdn.net/4.12/video.min.js"></script>
+<link href="//vjs.zencdn.net/5.1.0/video-js.min.css" rel="stylesheet">
```

Comment:
Can you add a thing linking to http://videojs.com/getting-started/ saying that you can always get the latest URL there?